pyiceberg/catalog/dynamodb.py

```diff
@@ -110,7 +118,7 @@ def _dynamodb_table_exists(self) -> bool:
             return False
         except self.dynamodb.exceptions.InternalServerError as e:
             raise GenericDynamoDbError(e.message) from e
-
+        print(response["Table"]["TableStatus"])
```

Comment:
Nit: could you please remove this print message?

Reply:
Not a nit at all, thanks for the catch